### ISSUE TITLE
Naming convention for atoms

### ISSUE THREAD

**Issue by hernan3009** [mdtraj/mdtraj#1305]
Hello,

I made a simulation of ethanol molecules, when I process it with MDTraj:

```
for at in somename.topology.residue(0).atoms:
    print(at.name)
```
I get names exactly as they appear in the geometry file. While the same code on a water (TIP3) simulation gives 'O', 'H1' and 'H2', instead of the names in my geometry file ('OW', 'HW1' and 'HW2'). This seems to be OK because the new names are reasonable.

I wish to know which is the name convention used by MDTraj.

Thank you for your work and time

Regadrs

**Comment by peastman** [comment]
Generally it uses PDB 3.0 names for atoms and residues.  In some cases you can override that and have it keep whatever names were in your input file, such as by specifying `standard_names=False` when calling `load_pdb()`.

**Comment by hernan3009** [comment]
Thank you for your reply. Excuse me, I did not know about `load_pdb`. I loaded the trajectory using `md.iterload`, it is still possible to use something similar to `standard_names=False` ? (it seems like not according to http://mdtraj.org/1.9.0/api/generated/mdtraj.iterload.html )

**Comment by peastman** [comment]
It doesn't look like it has that option, though presumably it could be added.

**Comment by mpharrigan** [comment]
You can pre-load the topology with load_pdb (using `standard_names`) and pass that topology to iterload instead of passing a filename

**Comment by peastman** [comment]
I thought about suggesting that.  The problem is that `load_pdb()` will load the entire file.  If you're using `iterload()`, presumably it's because it's a really big file and you don't want to load the whole thing into memory all at once.

But then I looked up the implementation of `iterload()`, and it's not clear to me whether it actually avoids that.  For PDB files, it looks like it may just go ahead and load the whole file all at once anyway.  If that's true (can someone else confirm that?), there's no advantage over just calling `load_pdb()`.

**Comment by mpharrigan** [comment]
The typical use case is a topology stored in a pdb (1 frame) and a long trajectory stored in something that lacks the topology like xtc

**Comment by hernan3009** [comment]
It seems to me more a design decision. MDTraj is great, but does not solve every MD problem (which is of course impossible). 

As a new user, I need to made my own routines (for solving my specifics issues). I think that it would be interesting to be able to have some freedom in the name convention (I made a dirty hack that works for my purposes). My specific problem was the I obtain an atom property from outside MDTraj (naming atoms according to my geometry file) and then I needed to link these values to atoms from MDTraj. As they have different names this was not direct.
